sandbox/src/mocks/retrieveReferralRequest/responses/cancelled.json

```diff
@@ -0,0 +1,294 @@
+{
+    "id": "000000070010",
+    "meta": {
+      "versionId": "5",
+      "profile": [
+        "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-ReferralRequest-1"
+      ]
+    },
+    "resourceType": "ReferralRequest",
+    "extension": [
+      {
+        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-Appointment-1",
+        "valueReference": {
+          "reference": "#appointment"
+        }
+      },
+      {
+        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ClinicalInfoFirstSubmitted-1",
+        "valueDateTime": "2021-06-11T12:33:23.167Z"
+      },
+      {
+        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralPriority-1",
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-Priority-1",
+              "code": "ROUTINE",
+              "display": "Routine"
+            }
+          ]
+        }
+      },
+      {
+        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralShortlist-1",
+        "valueReference": {
+          "reference": "#shortlist"
+        }
+      },
+      {
+        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-Commissioning-Rule-Org-1",
+        "valueIdentifier": {
+          "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+          "value": "R69"
+        }
+      },
+      {
+          "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralState-1",
+          "extension": [
+              {
+                  "url": "state",
+                  "valueCoding": {
+                      "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1",
+                      "code": "APPOINTMENT_CANCELLED_BY_PATIENT",
+                      "display": "Appointment Cancelled by Patient"
+                  }
+              },
+              {
+                  "url": "reason",
+                  "valueCoding": {
+                      "system": "__baseUrl__/FHIR/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON",
```

Comment:
This should be a request cancellation reason on the `ReferralRequest` yup, will update this now.

Reply:
Actually look at this again the defining state relates to the `Appointment` being cancelled rather than the referral, so I believe an `APPOINTMENT-CANCELLATION-REASON` is correct in this case.